src/test/resources/dto/common/driving.json

```diff
@@ -1,4 +1,5 @@
 {
   "type" : "driving",
-  "disable_border_crossing" : false
+  "disable_border_crossing" : false,
+  "traffic_model" : "optimistic"
```

Comment:
Can we add this as a separate file? I'd be happy if we could unit test both "driving transportation mode: defaults" and "driving transportation mode: explicit"